registry/registry.go

```diff
@@ -221,7 +222,7 @@ func setDirectoryURL(directoryurl string) *acme.Client {
 }
 
 // ListenAndServe runs the registry's HTTP server.
-func (registry *Registry) ListenAndServe() error {
+func (registry *Registry) ListenAndServe() (err error) {
```

Comment:
Can we rename the output variable? Using `err` for this can be a bit of a foot-gun, as it's very easy to accidentally shadow the variable.

I know in some other code-bases, we commonly used `retErr` for this, which makes it more clear that it's the output variable.

Reply:
FTR: there a boatload of these sprinkled throughout the whole codebase https://github.com/search?q=repo%3Adistribution%2Fdistribution+%22err+error%29%22&type=code